libbeat/template/template.go

```diff
@@ -277,7 +277,7 @@ func buildIdxSettings(ver common.Version, userSettings common.MapStr) common.Map
 		indexSettings.Put("number_of_routing_shards", defaultNumberOfRoutingShards)
 	}
 
-	if ver.Major >= 7 {
+	if ver.Major >= 6 {
```

Comment:
This enables it for all 6.x ES versions. Would it be better if we check the minor version as well?

```
ver.Major >= 7 || (ver.Major ==6 && ver.Minor >= 7)
```

Reply:
Should have pushed by local changes. My take at the moment is that it should set it for 6.x versions except 6.0.